src/test/ensemble_config_change_stress.cpp

```diff
@@ -1009,16 +1008,14 @@ TEST_F(StressPipelineConfigChanges, AddNewVersionDuringGetMetadataLoad) {
     std::set<StatusCode> allowedLoadResults = {};
     performStressTest(
         &StressPipelineConfigChanges::triggerGetPipelineMetadataInALoop,
-        &StressPipelineConfigChanges::defaultVersionRemove,
+        &StressPipelineConfigChanges::defaultVersionAdd,
         performWholeConfigReload,
         requiredLoadResults,
         allowedLoadResults);
 }
 TEST_F(StressPipelineConfigChanges, RemoveDefaultVersionDuringGetMetadataLoad) {
     std::set<StatusCode> requiredLoadResults = {StatusCode::OK,
-        StatusCode::PIPELINE_DEFINITION_NOT_LOADED_YET,  // we hit when all config changes finish to propagate
-        StatusCode::MODEL_VERSION_NOT_LOADED_ANYMORE,    // we hit default version which is unloaded already but default is not changed yet
```

Comment:
Why do we remove `MODEL_VERSION_NOT_LOADED_ANYMORE` and `MODEL_MISSING`?

Reply:
We do not hit those statuses since inputs info & outputs info are stored in PipelineDefinition.